ADOPTERS.md

```diff
@@ -14,6 +14,8 @@ A non-exhaustive unordered list of organizations that have adopted OpenTelemetry
 
 [Jaeger](https://jaegertracing.io) backend binaries are built on top of OpenTelemetry collector ([blog post](https://medium.com/jaegertracing/jaeger-embraces-opentelemetry-collector-90a545cbc24)).
 
+[Lightstep](https://lightstep.com)
```

Comment:
optional: add a few words on the usage scenarios of the SDK.

Reply:
i've added words